src/Uno.UI/Generated/3.0.0.0/Windows.UI.Xaml.Controls/Control.cs

```diff
@@ -539,7 +539,7 @@ protected virtual void OnPointerCaptureLost( global::Windows.UI.Xaml.Input.Point
 		}
 		#endif
 		// Skipping already declared method Windows.UI.Xaml.Controls.Control.OnPointerCanceled(Windows.UI.Xaml.Input.PointerRoutedEventArgs)
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __MACOS__
+		#if false || false || false || false || false
 		[global::Uno.NotImplemented]
```

Comment:
On all paltforms ??

Reply:
Not all platforms, but there is no point to raise an exception here, so I preferred to not add noisy `#if __WASM__` in code.